Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapper.kt

```diff
@@ -3,18 +3,22 @@ package de.rki.coronawarnapp.appconfig.mapping
 import dagger.Reusable
 import de.rki.coronawarnapp.appconfig.CWAConfig
 import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid.ApplicationConfigurationAndroid
-import de.rki.coronawarnapp.server.protocols.internal.v2.AppFeaturesOuterClass
+import de.rki.coronawarnapp.storage.TestSettings
+import de.rki.coronawarnapp.util.CWADebug
 import timber.log.Timber
 import javax.inject.Inject
 
 @Reusable
-class CWAConfigMapper @Inject constructor() : CWAConfig.Mapper {
+class CWAConfigMapper @Inject constructor(
+    private val testSettings: TestSettings
+) : CWAConfig.Mapper {
+
     override fun map(rawConfig: ApplicationConfigurationAndroid): CWAConfig {
         return CWAConfigContainer(
             latestVersionCode = rawConfig.latestVersionCode,
             minVersionCode = rawConfig.minVersionCode,
             supportedCountries = rawConfig.getMappedSupportedCountries(),
-            appFeatures = rawConfig.mapAppFeatures()
+            isDeviceTimeCheckEnabled = !rawConfig.isDeviceTimeCheckDisabled()
```

Comment:
minor: as far as I could see, in all other instances the disabled case (`isDeviceTimeCheckDisabled`)was used for all other booleans - why switch to enabled `isDeviceTimeCheckEnabled `here?

Reply:
Here we evaluate whether the config contains the "disable the check flag".
It doesn't tell us whether it should be enabled (which is also affected by other things like caching) it just tells us if we should disable it.

I use "isDeviceTimeCheckEnabled" in all other cases because there we can make the decision whether the check is enabled or not.